aggregator/src/binary_utils.rs

```diff
@@ -189,8 +190,15 @@ pub async fn datastore<C: Clock>(
 
 /// Loads a series of certificates from a PEM file into a rustls [`RootCertStore`].
 fn load_pem_trust_store(path: impl AsRef<Path>) -> Result<RootCertStore, io::Error> {
-    let mut buf_read = BufReader::new(File::open(path)?);
-    let der_certs = rustls_pemfile::certs(&mut buf_read).collect::<Result<Vec<_>, _>>()?;
+    let pem_data = fs::read(path)?;
+    let der_certs = CertificateDer::pem_slice_iter(&pem_data)
+        .collect::<Result<Vec<_>, _>>()
+        .map_err(|err| {
+            io::Error::new(
```

Comment:
We could just change the signature of this function to return `rustls_pki_types::pem::Error` instead. We already apply an anyhow "context" string to the error at the sole callsite.